symfony/client/src/orangehrmAdminPlugin/pages/employmentStatus/EditEmploymentStatus.vue

```diff
@@ -0,0 +1,135 @@
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text tag="h6">Edit Employment Status</oxd-text>
+
+      <oxd-divider />
+
+      <oxd-form @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-input-field
+              label="Employment Status Name"
+              v-model="employmentStatus.name"
+              :rules="rules.name"
+          />
+        </oxd-form-row>
+
+        <oxd-divider />
+
+        <oxd-form-actions>
+          <oxd-button
+              type="button"
+              displayType="ghost"
+              label="Cancel"
+              @click="onCancel"
+          />
+          <oxd-button
+              class="orangehrm-left-space"
+              displayType="secondary"
+              label="Update"
+              type="submit"
+          />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {navigate} from '@orangehrm/core/util/helper/navigation';
+
+export default {
+  props: {
+    employmentStatusId: {
+      type: Number,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      employmentStatus: {
+        id: '',
+        name: '',
+      },
+      rules: {
+        name: [],
+      },
+      errors: [],
+    };
+  },
+  methods: {
+    onSave() {
+      // TODO: Loading
+      this.$http
+          .put(`api/v1/admin/employment-statuses/${this.employmentStatus.id}`, {
+            name: this.employmentStatus.name,
+          })
+          .then(() => {
+            // go back
+            this.onCancel();
+          })
+          .catch(error => {
+            console.log(error);
+          });
+    },
+    onCancel() {
+      navigate('/admin/employmentStatus');
+    },
+  },
+  created() {
+    this.$http
+        .get(`api/v1/admin/employment-statuses/${this.employmentStatusId}`)
+        .then(response => {
+          const {data} = response.data;
+          this.employmentStatus.id = data.id;
+          this.employmentStatus.name = data.name;
+          // Fetch list data for unique test
+          this.$http.get(`api/v1/admin/employment-statuses`).then(response => {
+            const {data} = response.data;
+            this.rules.name.push(v => {
+              return (!!v && v.trim() !== '') || 'Required';
+            });
+            this.rules.name.push(v => {
+              return (
+                  (v && v.length <= 50) || 'Should be less than 50 characters'
```

Comment:
Character length exceeded message should follow this.

`Should not exceed #yourlimit# characters`